src/components/BookForm.jsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable */
+/* eslint-disable import/no-duplicates */
```

Comment:
- [ ] Just as I pointed out above, I'll suggest you remove these linters disable rule and try to fix the errors. :+1: :small_airplane: :smile: